packages/api/taskOrderFiles/createTaskOrderFile.ts

```diff
@@ -0,0 +1,66 @@
+import {APIGatewayProxyEvent, APIGatewayProxyResult} from "aws-lambda";
+import {v4 as uuid} from "uuid";
+import {ApiSuccessResponse, ErrorResponse, ErrorStatusCode, SuccessStatusCode} from "../utils/response";
+import {PutObjectCommand, S3Client} from "@aws-sdk/client-s3";
+import {FileMetadata, FileScanStatus} from "../models/FileMetadata";
+import * as parser from "lambda-multipart-parser"
+import {ErrorCodes} from "../models/Error";
+
+const bucketName = process.env.PENDING_BUCKET;
+
+/**
+ * Creates a new Task Order File
+ *
+ * @param event - The POST request from API Gateway
+ */
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+    const key = uuid();
+    const client = new S3Client({});
+
+    try {
+        const result = await parser.parse(event);
+        if (result.files.length != 1) {
+            return new ErrorResponse(
+                { code: ErrorCodes.INVALID_INPUT, message: "Expected exactly one file." },
+                ErrorStatusCode.BAD_REQUEST
+            );
+        }
+        else {
+            let file = result.files[0];
+            if (file.contentType != 'application/pdf') {
+                return new ErrorResponse(
+                    { code: ErrorCodes.INVALID_INPUT, message: "Expected a PDF file" },
+                    ErrorStatusCode.BAD_REQUEST
+                );
+            }
+            else {
+                const command = new PutObjectCommand({
+                    Bucket: bucketName,
+                    Key: key,
+                    Body: file.content,
+                    ContentType: file.contentType
+                });
+                const response = await client.send(command);
+                console.log(response);
```

Comment:
We probably don't actually want to do this?

Reply:
yep, removed